demos/sweep/main.py

```diff
@@ -39,14 +39,15 @@ def stop(self):
         """Reset the state of the demo if needed, else leave blank"""
         pass
 
-    def draw_vline_loc(self, x, val):
+    def draw_vline_loc(self, x, value):
         """
         Draw a vertical line at location x with value val
 
         Args:
             x (int): x location of the line
             val (int): value of the line
```

Comment:
Please update docstring param to match the signature of the function (i.e. val -> value)